### PR TITLE
15475 - Special Resolution UX fixes

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:* /bcgov/entity#15474

*Description of changes:*
https://preview.uxpin.com/53053d7aeb85fb29915f7afa596429aea4f2a964#/pages/162999485/simulate/sitemap

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/YourCompany/EntityName.vue [hunk 1]
```diff
@@ -16,7 +16,10 @@
         <label :class="{'error-text': invalidSection}">
           <strong>{{ getResource.entityReference }} Name</strong>
         </label>
-        <v-col md="1">
+        <v-col
+          md="1"
+          class="pa-0"
```
Comment (seeker25) [inline src/components/common/YourCompany/EntityName.vue:21]: This looked fine for SR - I did a Change of Name, double checking the other entity types now

**Comment by severinbeauvais** [comment]
I'm out of the office this week. Let me know if you really, really need me anyways.

**Review by JazzarKarim — APPROVED** [review]
Looks good to me! 👍

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-edit-dev--pr-503-wzv5b0u9.web.app

**Pull request merged**